### PR TITLE
[IA-3146][IA-3065] Updating to tf 2.7 gpu image

### PR DESCRIPTION
**Pull request by xuscrbw**

### CODE REVIEW

**Comment by louie-tsai** [comment]
added two commits to enable oneDNN on CPU.
please let me know once there is test env for vaildation. I could do some manually testing if needed.

**Comment by deflaux** [comment]
Can you folks please remove the package version pin for keras that I added via https://github.com/DataBiosphere/terra-docker/pull/286? Its needed for tensorflow 2.6, but not for tensorflow 2.7. I also added a corresponding comment to terra-jupyter-base/Dockerfile which can be deleted.

Or let me know if I may send a commit to this PR with that change. Thanks!

Also note that https://github.com/DataBiosphere/terra-docker/pull/286 fixes the test failure you are seeing here for `Test terra-jupyter-python`.

**Comment by xuscrbw** [comment]
@louie-tsai Hi! I had to re-add your branch into this one. I accidentally overwrote it before. It looks like the tests passed, so could you give it a look to make sure we aren't missing anything that you wanted to add?

File: terra-jupyter-base/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
 #######################
 # Prerequisites
 #######################
 ENV DEBIAN_FRONTEND noninteractive
 
+RUN sudo -i \
+    echo "deb http://security.ubuntu.com/ubuntu/ bionic-security main" >> /etc/apt/sources.list \
+    sudo apt update && apt-cache policy libssl1.0.0
+
+#RUN sudo -i \
```
Comment (Qi77Qi) [inline terra-jupyter-base/Dockerfile:16]: can we remove commnected out code

File: terra-jupyter-base/Dockerfile [hunk 2]
```diff
@@ -99,6 +105,7 @@ RUN pip3 -V \
  && pip3 install jupyter_contrib_nbextensions \
  && pip3 install jupyter_nbextensions_configurator \
  # for jupyter_delocalize.py and jupyter_notebook_config.py
+ # && pip3 install tornado==5.1.1 \
```
Comment (Qi77Qi) [inline terra-jupyter-base/Dockerfile:108]: remove

File: terra-jupyter-base/Dockerfile [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
+FROM gcr.io/deeplearning-platform-release/tf-gpu.2-7
+
 USER root
 
 #######################
 # Prerequisites
 #######################
 ENV DEBIAN_FRONTEND noninteractive
 
+RUN sudo -i \
```
Comment (Qi77Qi) [inline terra-jupyter-base/Dockerfile:12]: what is this for?

**Comment by louie-tsai** [comment]
> 

@Qi77Qi 
The PR looks good to me. thanks for adding the previous PR back.

File: terra-jupyter-aou/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
+## 2.0.10 - 2022-01-06T18:21:41.240337Z
+
+- Update `terra-jupyter-base` to `1.0.3`
+  - Bumping Google Deeplearning image to 2.7, removing gcc-6 and gcc-8 install due to package problems
```
Comment (Qi77Qi) [inline terra-jupyter-aou/CHANGELOG.md:4]: did you check if it already comes with the base image?

**Review by Qi77Qi — APPROVED** [review]

**Pull request merged**